src/Utility /utility.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { toast } from 'react-toastify';
+import "react-toastify/dist/ReactToastify.css";
+
+const NotificationWarn = (msg) => {
+   toast.warn(msg, {
```

Comment:
@RajanTank this is repeating code create one const and reuse it
`{
      autoClose: 3000,
      closeButton: false,
      position: toast.POSITION.TOP_RIGHT
   }`